packages/network/package.json

```diff
@@ -9,9 +9,19 @@
   "license": "MIT",
   "type": "module",
   "exports": {
-    ".": "./dist/index.js"
+    ".": "./dist/index.js",
+    "./dev": "./dist/dev/index.js"
+  },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "src/index.ts"
+      ],
+      "dev": [
+        "./src/dev/index.ts"
+      ]
```

Comment:
in other packages we used `"index":` for the index types instead of `*` - do you know what's the difference / if there is a difference? Should we standardise on one of them everywhere?

Also the `src/index.ts` path is not prefixed with `./` while the others are - is that intended?

Reply:
I think I copied this pattern from elsewhere, so we should probably make all of these consistent so future copy+paste is easier to get right.